### ISSUE TITLE
ACCESSIBILITY_TWEAKS break intended product listing filter design

### ISSUE THREAD

**Issue by cramytech** [shopware/platform#10218]
### Shopware Version

6.6.10.4

### Affected area / extension

Platform(Default)

### Actual behaviour

Text is not vertical centered anymore.
Border on hover is missing.

<img width="604" alt="Image" src="https://github.com/user-attachments/assets/d9c19c8a-2a20-4b7a-9ae2-92916842a3f2" />

### Expected behaviour

Intended design.

### How to reproduce

ACCESSIBILITY_TWEAKS=1

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Developer documentation written/updated.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.

**Comment by tobiasberge** [comment]
Hi @cramytech,

you need to run `theme:compile` once after activating the `ACCESSIBILITY_TWEAKS` feature flag. The flag cannot be toggled fully in runtime. 

The reason is that there are also adjustments within the SCSS (standard font-size, refactored styling etc.) that also use `ACCESSIBILITY_TWEAKS` feature flag checks.

Before `theme:compile` (Similar to your screenshot)
![Image](https://github.com/user-attachments/assets/01665cb5-f4f1-4e20-ac73-0bf9ec7be9d7)

After `theme:compile` (Expected styling)
![Image](https://github.com/user-attachments/assets/89d73d48-8a61-4de6-98ef-bab685a38fe1)

(Tested with v6.6.10.4)

Therefore, we would close this as cannot reproduce.

**Comment by cramytech** [comment]
My bad, sorry. Thanks a lot for the explanation.
